models/reviewModel.js

```diff
@@ -8,3 +8,15 @@ exports.findOneReview = (id) => {
 
   return db.query(query, [id]).then((res) => res.rows);
 };
+
+exports.findAllReviews = () => {
+  const queryString = `
+  SELECT reviews.*, CAST(COUNT(comments.comment_id) AS INT) AS comment_count 
+  FROM reviews
+  LEFT JOIN comments ON comments.review_id = reviews.review_id
+  GROUP BY reviews.review_id
+  ORDER BY created_at DESC;
+  `;
+
```

Comment:
Great stuff James, can you update your query so it's only sending the keys that the ticket is asking for - i.e. it shouldn't be sending the `body` column